### PR TITLE
2111.2 release notes

### PR DESCRIPTION
**Pull request by sptramer**
## Change summary

Release notes for version 2111.2 (scheduled for 1/26)

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/release-notes/archive/2111-2-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+Open 3D Engine release 2111.2 is a maintenance and quality of life improvement release based on 2111.1. This release is bugfix-only and contains no new features.
+
+## Bug fixes
+
+* Duplicate engine detection and help in Project Manager [#6984](https://github.com/o3de/o3de/pull/6984)
+* Add MainPipeline to pass hierarchy for when HDR color grading component requests for the lut generation's image attachment. [#6436](https://github.com/o3de/o3de/pull/6436)
+* fixes for stability [#6434](https://github.com/o3de/o3de/pull/6434) 
+* Fixed SRG updated related crash
```
Comment (Ulrick28) [inline content/docs/release-notes/archive/2111-2-release-notes.md:16]: This item and the next should be indented to show it is a subset of the PR above. Any items without a PR are subsets of the PR above them.

Comment (sptramer) [reply content/docs/release-notes/archive/2111-2-release-notes.md]: Fixed; formatting error.

**Review by forhalle — APPROVED** [review]

**Review by forhalle — APPROVED** [review]
approving again after date change

**Review by Ulrick28 — APPROVED** [review]
Looks good, thanks for the edit on the known issues!

File: content/docs/release-notes/archive/_index.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ Looking for older versions of release notes for **Open 3D Engine (O3DE)**? Look
 
 | Version     |  Link                                           | Date of Release    |
 |-------------|-------------------------------------------------|--------------------|
-| 2107.1      | [2111.1 Release Notes](./21-11-release-notes)     | 12/2/2021          |
+| 2111.2      | [2111.2 Release Notes](./2111-2-release-notes)  | 1/27/2022 |
+| 2111.1      | [2111.1 Release Notes](./21-11-release-notes)     | 12/2/2021          |
 | 2107.1      | [2107.1 Release Notes](./2107-1-release-notes)   | 7/6/2021           |
```
Comment (willihay) [inline content/docs/release-notes/archive/_index.md:16]: These dates of release may be confusing or misinterpreted by people who are used to DD/MM/YYYY format. To be more inclusive, consider switching to something more explicit, like "6 Jul 2021" and "2 Dec 2021".

**Review by willihay — APPROVED** [review]

**Pull request merged**